### ISSUE TITLE
[Bug] Shape Manager menu broken

### ISSUE THREAD

**Issue by autive42** [maruohon/minihud#153]
Whenever I create a shape in the shape manager menu, the new shape displays over the add new shape controls.
![Screenshot of the issue](https://i.imgur.com/DYf31Sr.jpg)
It is still usable, but you have to carefully position your mouse to only hit the button you want.

**Comment by maruohon** [comment]
This menu being messed up is a known issue. I haven't bothered fixing it in this old code base, as most of the menu code has been rewritten already in the main development code, which will replace the current versions of the mod once it's ready to be ported. This mostly happens if you use the Large GUI scale in vanilla settings (I think it's called scale 3 now?).

The menus are mostly designed for scale 2 at 1280x720 resolution, which is what I run the game at. With scale 3 most things don't fit on screen nicely and stuff starts to wrap around and overlap. But this should also be mostly fixed already in the development code, because that top bar of buttons does not get split to multiple rows anymore, but instead it will have left and right arrow cycle buttons if needed. Plus thanks to the other malilib improvements many of the "unnecessary" config categories have been removed/combined into one, such as the three separate Info Line related categories merging into one, which decreases the number of category buttons needed.

So basically the buggy menus will get fixed once I get all the development code ported to all the MC versions. But no idea yet when it will be ready... For now, if possible (if it's not too small for your eyes), I'd recommend switching the vanilla GUI scale to 2. The new malilib does/will also have a separate option to switch the scaling just for my client mods so that you don't need to change the "global" vanilla scale value which affects all screens.

**Comment by autive42** [comment]
Changing my scale settings from auto to 3 fixed it for my screen. The auto setting seemed to be putting it at 4, which was the largest option and the only one where this was broken.

Your mods are great by the way. They greatly improve my Minecraft experience and I can't imagine going back to playing without them. Keep up the good work!

**Comment by maruohon** [comment]
Thanks :)

**Comment by Jkosio23** [comment]
Just another instance of this for me, on my main screen, GUI Scale 3 is large enough to just barely push the Shapes category button down to the second line, which seems like it's a key factor of this bug
